packages/ui/components/form-core/src/validate/ValidateMixin.js

```diff
@@ -685,7 +686,7 @@ export const ValidateMixinImplementation = superclass =>
       }
 
       for (const validatorToSetup of this._allValidators) {
-        if (!(validatorToSetup instanceof Validator)) {
+        if (validatorToSetup.type === undefined) {
```

Comment:
I am not entirely sure why initially the check was added but it looks like when validators are setup, we need to ensure that passed object is valid Validator(s).

Reply:
The goal indeed is to make sure that we pass an instance of a Validator.
With this change, every object with a `type` property would pass (for instance an `<input>` and this is not what we want :)

With multiple versions of Lion, the instanceof check will fail. Lit faced the same issue and they solved it with: https://github.com/lit/lit/blob/main/packages/lit-html/src/lit-html.ts#L1468
 
They check for value[['_$litType$']] instead of value_$litType$ as the latter can be renamed during build/minification.

So the goal is to add a marker that uniquely identifies it as a Validator, that works across multiple lion versions.

Maybe you can add `['_$isValidator$']` to `Validator` class?

Reply:
Ah, right. I tried playing around adding a new prop but it seemed redundant. But it's indeed a good catch that `<input>` would also be considered as a validator with current changes. Let me add a new prop instead. :)

Reply:
Awesome. Would it also be possible to add one or two tests that otherwise would have failed?

Reply:
And after this the same change should be applied to the lit2 branch: https://github.com/ing-bank/lion/tree/lit2

Reply:
@tlouisse, can you take a look? I will add the changeset log and cherry-pick the changes for lit2.

Reply:
Thanks 👏 
Did a few changes (like moving it to the constructor to be a bit more unobtrusive). Please have a look. 
Would be great if you want to bring this to the lit2 branch as well

Reply:
Thanks,  I will cherry-pick and make a new PR after this is merged. :)